src/utils/object.js

```diff
@@ -56,7 +56,7 @@ export function dict(opt_initial) {
  * Checks if the given key is a property in the map.
  *
  * @param {T}  obj a map like property.
- * @param {string}  key
+ * @param {?string}  key
```

Comment:
This will throw runtime errors. Was this just because CC was giving type errors?

Reply:
Oh man, I thought we were changing it to allow nullable `obj`, not the `key`. This is fine.